x-pack/plugins/metrics_entities/server/routes/delete_transforms.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import { IRouter } from '../../../../../src/core/server';
+import { METRICS_ENTITIES_TRANSFORMS } from '../../common/constants';
+import { ModuleNames } from '../modules';
+
+import { getMetricsEntitiesClient } from './utils/get_metrics_entities_client';
+
+/**
+ * Deletes transforms.
+ * NOTE: We use a POST rather than a DELETE on purpose here to ensure that there
```

Comment:
👍 Thanks for the note here -- especially with the history we've had with issues around using DELETE w/ body.